spec/system/admin_manage_auth_settings_spec.rb

```diff
@@ -8,11 +8,16 @@
   let(:admin) { create(:user, :admin, :confirmed, organization: organization) }
   let(:auth_settings) { create(:auth_setting, organization: organization) }
   let(:decidim_half_signup_admin) { Decidim::HalfSignup::AdminEngine.routes.url_helpers }
+  let(:sms_gateway_service) { "Decidim::Verifications::Sms::ExampleGateway" }
 
   before do
     sign_in admin
     switch_to_host(organization.host)
     visit decidim_admin.edit_organization_path
+
+    Decidim.configure do |config|
```

Comment:
Prefer to define mock for `Decidim.configure.sms_gateway_service` 

Example : `allow(Decidim.config.sms_gateway_service).to receive(:present?).and_return(true)`